internal/fnruntime/runner.go

```diff
@@ -111,10 +112,11 @@ func (fr *FunctionRunner) Filter(input []*yaml.RNode) (output []*yaml.RNode, err
 	if !fr.disableCLIOutput {
 		pr.Printf("[RUNNING] %q\n", fr.name)
 	}
+	t0 := time.Now()
 	output, err = fr.do(input)
 	if err != nil {
 		printOpt := printer.NewOpt()
-		pr.OptPrintf(printOpt, "[FAIL] %q\n", fr.name)
+		pr.OptPrintf(printOpt, "[FAIL] %q in %v\n", fr.name, time.Since(t0).Truncate(time.Millisecond*100))
```

Comment:
curious the reasoning for rounding to 100ms here ? I was playing with the default formatter %v for time, works well for this case https://play.golang.org/p/0ZcLEAwmuGb

Reply:
Looks great for artificial inputs, but real world inputs will be much worse. For example, https://play.golang.org/p/asojWEV2kBc you end up with things like `12.783198789s`. If we are getting execution down to a lot less than a second it would make sense to drop this to 1/10ms probably, but right now everything is in the seconds range. So I thought `4.9s` is sufficient enough. I don't mind `4.92s` or `4.923s` though - but `12.783198789s` is a bit excessive :slightly_smiling_face:

Reply:
> but real world inputs will be much worse.

Aah. Thats what I was missing. I think 100ms truncation is reasonable.